lms/static/sass/_build-lms-v2.scss

```diff
@@ -3,7 +3,22 @@
 // Version 2 - introduces the Pattern Library
 
 
-// Configuration
```

Comment:
@andy-armstrong and gang, hey!!! The fort is getting bigger. Working on some really fun stuff around Font Awesome now, including a11y updates. Will spam you guys with updates soon.

*Going to watch that teaser now* :)

Hope you all are well. Miss ya!